sdk/cosmosdb/cosmos/src/utils/tracing.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createTracingClient } from "@azure/core-tracing";
+import { Constants } from "../common/constants";
+
+/**
+ * Creates a span using the global tracer.
+ *
+ * @param name - The name of the operation being performed.
+ * @param tracingOptions - The options for the underlying http request.
+ *
+ * @internal
+ */
```

Comment:
This comment seems to describe a function rather than an object. We used to use tracing functions with things like `name` and `tracingOptions`, so if you copied this definition from somewhere else, could you share where that was so that I can go correct it 😅 ?

Reply:
This is in your code samples.

Reply:
Here fix it ->  https://github.com/Azure/azure-sdk-for-js/issues/21199 😄 I should be on your PRs smh.